app/models/dashboard.rb

```diff
@@ -40,7 +41,20 @@ def closed_defence_requests
   attr_reader :user, :defence_requests_scoped_by_policy
 
   def ordered_defence_requests
-    defence_requests_scoped_by_policy.ordered_by_created_at
+    defence_requests_scoped_by_policy.ordered_by_created_at.map { |d| new_dash_dr[d] }
+  end
+
+  def new_dash_dr
+    @_new_dash_dr ||= DefenceRequest.method(:new).curry(2)[@client]
```

Comment:
Is there a reason we're using @_new_dash_dr rather than just @new_dash_dr ? I see we've done it in a few places, but I don't think it's necessary, right? We're not using C :) http://stackoverflow.com/a/25090719